pkgs/development/python-modules/jsbeautifier/default.nix

```diff
@@ -2,17 +2,19 @@
 
 buildPythonApplication rec {
   pname = "jsbeautifier";
-  version = "1.8.9";
+  version = "1.9.1";
 
   propagatedBuildInputs = [ six ];
 
-  buildInputs = [ EditorConfig pytest ];
+  buildInputs = [ EditorConfig ];
```

Comment:
This belongs into `propagatedBuildInputs`.

Reply:
EditorCOnfig should be in `propagatedBuildInputs`, pytest in `checkInputs`.

Reply:
@apeyroux Why was this marked as resolved? EditorConfig is still not in propagatedBuildInputs.

Reply:
sorry ...